core/platform/email/gae_email_services.py

```diff
@@ -79,14 +80,25 @@ def send_mail(
     if not mail.is_email_valid(recipient_email):
         raise ValueError(
             'Malformed recipient email address: %s' % recipient_email)
-
-    if bcc_admin:
-        mail.send_mail(
-            sender_email, recipient_email, subject, plaintext_body,
-            html=html_body, bcc=[feconf.ADMIN_EMAIL_ADDRESS])
-    else:
-        mail.send_mail(
-            sender_email, recipient_email, subject, plaintext_body,
-            html=html_body)
+    if feconf.EMAIL_SERVICE_PROVIDER == feconf.GAE_EMAIL_SERVICE:
```

Comment:
Also, do you think we can get rid of send_mail_to_admin() and use the general send_mail() instead wherever that is used? I think that would reduce duplication, and then we can keep this file for just the platform-level code. There's something to be said also for maybe moving some of the first few checks in send_mail() to the domain layer, but that's up to you and I'm happy either way.